src/hoc/withFeatureFlag.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from "react";
```

Comment:
I think this should be in `hocs` to match the pluralisation of `hooks`, `components` etc.